ssr-e2e/proxy-test.spec.js

```diff
@@ -0,0 +1,176 @@
+const httpProxy = require('http-proxy');
+const http = require('http');
+const fs = require('fs');
+
+const proxy = httpProxy.createProxyServer({ secure: false });
+
+// TODO: Use environment variable instead
+// const BACKEND_BASE_URL = process.env.OCC_URL;
+// const BACKEND_BASE_URL = 'https://40.76.109.9:9002';
+const BACKEND_BASE_URL = 'https://20.83.184.244:9002';
+
+const SSR_LOG_PATH = './ssr-e2e/ssr.log';
+
+const REQUEST_OPTIONS = {
+  host: 'localhost',
+  port: 4000,
+};
+
+describe('SSR E2E', () => {
+  let server;
+
+  beforeAll(async () => {
+    // Hacky way to make sure server starts
+    await new Promise((res) => setTimeout(res, 30000));
```

Comment:
As we discussed offline, it would be good to avoid big magic numbers for waiting until something happens. Instead it would be good to react to a specific event that we're interested in (e.g. the server actually started).